diff --git a/MountainLobster.cpp b/MountainLobster.cpp
index 74e07f1..4610dd6 100644
--- a/MountainLobster.cpp
+++ b/MountainLobster.cpp
@@ -33,7 +33,7 @@ int main(void)
 	initApp.physicsFps = 120;
 	initApp.width = 1920;//1024;
 	initApp.height = 1080;//768;
-	initApp.windowStyle = KWindowStyle::Windowed_Fixed_Size;
+	initApp.windowStyle = KWindowStyle::Windowed_Resizeable;
 	initApp.windowTitle = KTEXT("Demo");
 	StartupEngine(&initApp);
 
diff --git a/src/GameSetup.cpp b/src/GameSetup.cpp
index 43291e8..de43f47 100644
--- a/src/GameSetup.cpp
+++ b/src/GameSetup.cpp
@@ -377,7 +377,7 @@ void GameSetup::setBackgroundShaderParams()
 	m_pBackgroundShader->setUniformArray("planetPositions", &planetPositions[0], PLANETS_COUNT);
 	m_pBackgroundShader->setUniformArray("planetColours", &planetCols[0], PLANETS_COUNT);
 	
-	//int h = GET_APP()->getRenderWindow()->getView().getSize().y;
+	int h = GET_APP()->getRenderWindow()->getView().getSize().y;
 	m_pBackgroundShader->setUniform("windowHeight", (int)GET_APP()->getRenderWindow()->getView().getSize().y);
 	m_pBackgroundShader->setUniform("colScale", colScale);
 	
